index.html

```diff
@@ -0,0 +1,301 @@
+                            <li class="tag">Ruby on rails</li>
+                            <li class="tag">css</li>
+                            <li class="tag">JavScript</li>
+                            <li class="tag">html</li>
+                        </ul>
+                    </div>
+                    <div class="see-project">
+                        <button class="see-project">See Project</button>
+                    </div>
+                </div>
+            </div>
+
+            <div class="card flex-center">
+                <div class="img-multi"></div>
+                <div class="text flex-center">
+                    <h3>Multi-Post Stories <br>Gain+Glory</h3>
+                    <div class="tags">
+                        <ul class="flex-center">
+                            <li class="tag">Ruby on rails</li>
+                            <li class="tag">css</li>
+                            <li class="tag">JavScript</li>
+                            <li class="tag">html</li>
+                        </ul>
+                    </div>
+                    <div class="see-project">
+                        <button class="see-project">See Project</button>
+                    </div>
+                </div>
+            </div>
+
+            <div class="card flex-center">
+                <div class="img-multi"></div>
+                <div class="text flex-center">
+                    <h3>Multi-Post Stories <br>Gain+Glory</h3>
+                    <div class="tags">
+                        <ul class="flex-center">
+                            <li class="tag">Ruby on rails</li>
+                            <li class="tag">css</li>
+                            <li class="tag">JavScript</li>
+                            <li class="tag">html</li>
+                        </ul>
+                    </div>
+                    <div class="see-project">
+                        <button class="see-project">See Project</button>
+                    </div>
+                </div>
+            </div>
+
+            <div class="card flex-center">
+                <div class="img-multi"></div>
+                <div class="text flex-center">
+                    <h3>Multi-Post Stories <br>Gain+Glory</h3>
+                    <div class="tags">
+                        <ul class="flex-center">
+                            <li class="tag">Ruby on rails</li>
+                            <li class="tag">css</li>
+                            <li class="tag">JavScript</li>
+                            <li class="tag">html</li>
+                        </ul>
+                    </div>
+                    <div class="see-project">
+                        <button class="see-project">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="about mt">
+        <div class="contant-1 flex-all">
+            <h1 class="about-me">About me</h1>
```

Comment:
- [OPTIONAL]  It would be nice to change this to another `hx(h2-h6)` tag instead of the `h1` tag. Based on convention, there should be only one `h1` tag on a webpage. 💻